src/main/java/com/example/bookshop/config/SecurityConfig.java

```diff
@@ -1,23 +1,29 @@
 package com.example.bookshop.config;
```

Comment:
As mentioned in the previous PR, it is better to place it in the security package. Be careful when merging branches)